### ISSUE TITLE
[A11y] Command action in tree item should be able to be set according to use case

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#10287]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.71.1
- OS Version: Version: Windows_NT x64 10.0.22621

Steps to Reproduce:

In our extension, there're several actions on tree item, user can click the 'Learn more' action(outlined in red below) to go to browser for more details. Hence it should be a 'link' instead of a 'button'. Developers should be able to set the role for actions on tree items with VS Code provide an API for it. But now I'll not able to change the role, it's always 'button'
![image](https://user-images.githubusercontent.com/73154171/196641579-e091a971-9c4f-4758-af77-9cf89bdd8861.png)
